helm/ako/crds/ako.vmware.com_aviinfrasettings.yaml

```diff
@@ -91,6 +91,22 @@ spec:
                 type: object
                 required:
                 - shardSize
+              nsxSettings:
+                properties:
+                  project:
+                    type: string
+                  t1lr:
+                    type: string
+                type: object
+                required:
+                - t1lr
+              cloud:
```

Comment:
Can we remove `cloud` as we are not handling that from everywhere?

Reply:
done